main/website/index.html

```diff
@@ -0,0 +1,203 @@
+                    method: 'POST',
+                    body: new URLSearchParams(post_obj)
+                });
+                status_message = await response.text();
+                if (response.ok) {
+                    location.reload();
+                }
+            } catch (error) {
+                status_message = `ERROR: Couldn't post settings to server: ${error}`;
+            }
+        } else {
+            status_message = 'No request sent because forms weren\'t updated.';
+        }
+        " action=''>
+
+
+        <label for='eth_use_static'>
+            Use static IP for ethernet (disables DHCP):
+        </label>
+        <input type='checkbox' id='eth_use_static' x-model='conf.eth_use_static'>
+        <br><br>
+
+        <label for='eth_ip'>
+            Ethernet static IP (if enabled):
+        </label>
+        <input type='text' id='eth_ip' x-model='conf.eth_ip'>
+        <br><br>
+
+        <label for='eth_netmask'>
+            Ethernet netmask IP (if enabled):
+        </label>
+        <input type='text' id='eth_netmask' x-model='conf.eth_netmask'>
+        <br><br>
+
+        <label for='eth_gw'>
+            Ethernet gateway IP (if enabled):
+        </label>
+        <input type='text' id='eth_gw' x-model='conf.eth_gw'>
+        <br><br>
+
+
+
+        <label for='wifi_enabled'>
+            Wifi enabled:
+        </label>
+        <input type='checkbox' id='wifi_enabled' x-model='conf.wifi_enabled'>
+        <br><br>
+
+        <label for='wifi_ssid'>
+            WIFI SSID name:
+        </label>
+        <input type='text' id='wifi_ssid' x-model='conf.wifi_ssid'>
+        <br><br>
+
+        <label for='wifi_pass'>
+            WIFI password (not shown):
+        </label>
+        <input type='password' id='wifi_pass' x-model='conf.wifi_pass'>
+        <br><br>
+
+
+        <label for='wifi_use_static'>
+            Use static IP for WIFI (disables DHCP):
+        </label>
+        <input type='checkbox' id='wifi_use_static' x-model='conf.wifi_use_static'>
+        <br><br>
+
+        <label for='wifi_ip'>
+            WIFI static IP (if enabled):
+        </label>
+        <input type='text' id='wifi_ip' x-model='conf.wifi_ip'>
+        <br><br>
+
+        <label for='wifi_netmask'>
+            WIFI netmask IP (if enabled):
+        </label>
+        <input type='text' id='wifi_netmask' x-model='conf.wifi_netmask'>
+        <br><br>
+
+        <label for='wifi_gw'>
+            WIFI gateway IP (if enabled):
+        </label>
+        <input type='text' id='wifi_gw' x-model='conf.wifi_gw'>
+        <br><br>
+
+
+
+        <label for='can_bitrate'>
+            CAN Bitrate in kbits
+            (supported: 25, 50, 100, 125, 250, 500, 800, 1000):
+        </label>
+        <input type='number' id='can_bitrate' x-model='conf.can_bitrate'>
+        <br><br>
+
+
+        <input type='submit' value='Submit'>
+
+        <p style="background:#fff8ba;" x-text='status_message'></p>
```

Comment:
I'd recommend making a CSS class for the background style since you use it twice. Inline styles are also typically something to avoid if you subscribe to https://en.wikipedia.org/wiki/Separation_of_content_and_presentation.

Reply:
True. Fixed this in 72219c3.